sormas-e2e-tests/src/test/resources/features/sanity/web/CaseFilters.feature

```diff
@@ -12,4 +12,186 @@ Feature: Case filter functionality
     And I check that all displayed cases have "Not yet classified" in grid Case Classification column
     When I search for cases created with the API using Person's name
     Then I apply Disease filter "COVID-19"
-    And I check that all displayed cases have "COVID-19" in grid Disease column
\ No newline at end of file
+    And I check that all displayed cases have "COVID-19" in grid Disease column
+
+  Scenario: Check Cases Origin filter
```

Comment:
1. Add jira test case identifier
2. The test is too long, and if any step will fail, the rest of the checks won't work. Let's have the background for case creation and login, then a separated test to check each filter